src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -102,9 +102,14 @@ class PrefixModeStorage {
 
   bool is_package_exempted(const std::string& package);
 
+  // When using a proto package as the prefix, this should be added as the
+  // prefix in front of it.
+  const std::string forced_package_prefix() const { return forced_prefix_; }
```

Comment:
const& if you are trying to avoid the copy, or no const if you want to have the copy

Reply:
Added.